src/style.css

```diff
@@ -1,3 +1,68 @@
 body {
   margin: 0;
+  font-family: Roboto, sans-serif;
```

Comment:
Remember to use fallback fonts. If the main font doesn't load for some reason, the text will still be displayed in a decent way. Add a generic font family at the end of the list of fonts.